MaterialDesignThemes.Wpf/PaletteHelper.cs

```diff
@@ -10,7 +10,95 @@ namespace MaterialDesignThemes.Wpf
 {
     public class PaletteHelper
     {
-        public void SetLightDark(bool isDark)
+        #region Win32
+            [DllImport("dwmapi.dll", PreserveSig = false)]
+            private static extern bool DwmIsCompositionEnabled();
+
+            [DllImport("dwmapi.dll", EntryPoint = "#127")]
+            static extern void DwmGetColorizationParameters(ref DWMCOLORIZATIONPARAMS dp);
+
+            [StructLayout(LayoutKind.Sequential)]
+            public struct DWMCOLORIZATIONPARAMS
```

Comment:
Can this be private?  Don't want this to become part of our API.  Keeping it private will allow internal refactoring without changing our API.